### DISCUSSION TITLE
PROBLEMAS PARA GENERAR NUEVOS PERFILES O SUSCRIPCIONES

### DISCUSSION THREAD

**Discussion by CasP0** [community/community#147175] [Other Feature Feedback, Questions, & Ideas]
PORQUE NO SE ABREN NUEVOS PERFILES O ESTÁN QUEDANDO ESPACIOS EN BLANCO DENTRO DEL NÚCLEO DE SUBCARPETAS

**Comment by mecodeatlas** [comment]
Hi @CasP0, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
